cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -569,3 +569,53 @@ Feature: JWT Authenticator - Fetch signing key
     """
     cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
     """
+
+  Scenario: provider-uri with untrusted self sign certificate
+    Given I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/keycloak
+      body:
+      - !webservice
+
+      - !variable
+        id: provider-uri
+    """
+    And I am the super-user
+    And I successfully set authn-jwt "provider-uri" variable value to "https://jwks" in service "keycloak"
+    And I fetch an ID Token for username "alice" and password "alice"
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the ID token
+    Then the HTTP response status code is 502
```

Comment:
Add at the beginning of this test the bug number to make sure we will not forget it (mention that it suppose to be 401)